Scripts/Test-BranchName.ps1

```diff
@@ -59,10 +59,10 @@ function Write-FunctionAppName {
 
 # --- RegEx Patterns
 $V1MasterBranchRegEx = "^master$"
-$V1FeatureBranchRegEx = "^(?:CDS|SDD)-\d+-\w+[-v\d]{0}$"
+$V1FeatureBranchRegEx = "^(master)(-v\d+)?|(release\/).*|(CDS|SDD|NCSLT)(-\d+)(.*)(-v\d+){1}$"
 
```

Comment:
**note to self:** we need to ensure we correctly capture "-v1" or no version suffix names as the $v1 variable, and others need to set $v2orhigher or else ir results in release pipeline issues down the line